versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/reporting/AbstractVersionsReportRenderer.java

```diff
@@ -64,10 +64,6 @@
     protected ArtifactVersionsCache allUpdatesCache
             = new ArtifactVersionsCache( AbstractVersionDetails::getAllUpdates );
 
-    /**
```

Comment:
Removed this documentation comment as it was misleading (not used by Plexus) and not really necessary -- the parameters are pretty obvious.